packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -1593,7 +1593,7 @@ export class API extends EventEmitter {
   // * @param {String} opts.multisigContractAddress optional: MULTISIG ETH Contract Address
   // * @param {Callback} cb
   // */
-  getBalance(opts, cb) {
+  getBalance(opts, cb, baseUrl) {
```

Comment:
by default baseUrl should be the current URL (/v2/balance/ ? )

Reply:
ok

Reply:
Cool